ducktape/command_line/defaults.py

```diff
@@ -46,7 +46,8 @@ class ConsoleDefaults(object):
     # Log this to indicate a test is misbehaving to help end user find which test is at fault
     BAD_TEST_MESSAGE = "BAD_TEST"
 
-    # Test driver listens on this port for requests
-    TEST_DRIVER_PORT = 5556
-
+    # Ducktape will try to pick a random open port in the range [TEST_DRIVER_PORT_MIN, TEST_DRIVER_PORT_MAX]
+    # this range is *inclusive*
+    TEST_DRIVER_MIN_PORT = 5556
+    TEST_DRIVER_MAX_PORT = 5586
```

Comment:
This range seems small. 30 may be the # of hosts we use now, but I expect that number to get quite a bit larger.